src/app/components/validation-item-list/ValidationItem.jsx

```diff
@@ -26,8 +26,9 @@ export default class ValidationItem extends Component {
                     onClick={this.preventUserInteractions}
```

Comment:
I think you should have onChange even here

Reply:
This is a read-only checkbox so the user is not able to interact with it and so the onChange function will never be fired. :)

Reply:
so what is onClick doing there? :-)

Reply:
Yer the onClick was there to capture clicks then discard them, as otherwise users could click the checkbox to change its state. With the disabled attribute added as per another comment, this is no longer needed so has been removed

Reply:
Since moving to a disabled state the onClick event has been removed